protocol/0056-REWA-rewards_overview.md

```diff
@@ -98,25 +184,100 @@ Reward accounts and balances must be saved in [LNL checkpoints](./0073-LIMN-limi
+The entire reward account balance is paid out every epoch unless the total value of the metric over all entities is zero, in which case the balance will also be zero anyway (there are no fractional payouts).
 
-```math
-[p_1, m_1]
-[p_2, m_2]
-...
-[p_n, m_n]
+Rewards are first [distributed amongst entities](#distributing-rewards-amongst-entities) (individuals or teams) and then any rewards distributed to teams are [distributed amongst team members](#distributing-rewards-amongst-team-members).
+
+### Distributing rewards amongst entities
+
+Rewards are distributed amongst entities based on the distribution method defined in the recurring transfer.
+
+The protocol currently supports the following distribution strategies:
+
+- [pro-rata]:(#distributing-pro-rata) distributed pro-rata by reward metric
+- [rank]:(#distributing-based-on-rank) distributed by entities rank when ordered by reward metric
+
+#### Distributing pro-rata
+
+Rewards funded using the pro-rata strategy should be distributed pro-rata by each entities reward metric scaled by any active multipliers that party has, i.e.
+
+Let:
+
+- $d_{i}$ be the payout factor for entity $i$
+- $r_{i}$ be the reward metric value for entity $i$
+- $M_{i}$ be the sum of all reward payout multipliers for entity $i$ (reward payout multipliers include the [activity streak multiplier](./0086-ASPR-activity_streak_program.md#applying-the-activity-reward-multiplier) and [bonus rewards multiplier](./0085-RVST-rewards_vesting.md#determining-the-rewards-bonus-multiplier)).
+- $s_{i}$ be the share of the rewards for entity $i$
+
+$$d_{i}=r_{i} M_{i}$$
+
+Note if the entity is a team, $M_{i}$ is set to `1` as reward payout multipliers are considered later when distributing rewards [amongst the team members](#distributing-rewards-amongst-team-members).
+
+Calculate each entities share of the rewards, $s_{i}$ pro-rata based on $d_{i}$, i.e.
+
+$$s_{i} = \frac{d_{i}}{\sum_{i=1}^{n}d_{i}}$$
+
+#### Distributing based on rank
+
+Rewards funded using the rank-distribution strategy should be distributed as follows.
+
+1. Calculate each entity's reward metric.
+2. Arrange all entities in a list in descending order based on their reward metric values and determine their rank. If multiple entities share the same reward metric value, they should be assigned the same rank. The next entity's rank should be adjusted to account for the shared rank among the previous entities. For instance, if two entities share rank 2, the next entity should be assigned rank 4 (since there are two entities with rank 2).
+3. Set the entities `share_ratio` based on their position in the `rank_table` specified in the recurring transfer.
```

Comment:
Is there a need for a param for maximum rank table size? If not we should specify the mac size here. I guess it could be pretty big like a few hundred or low thousands entries without issue… can't imagine anyone want tons of entries really.